app/controllers/v0/disability_compensation_forms_controller.rb

```diff
@@ -54,6 +54,13 @@ def suggested_conditions
       render json: DisabilityContentionSerializer.new(results)
     end
 
+    def add_0781_metadata(form526)
+      if form526['syncModern0781Flow'].present?
+        { sync_modern0781_flow: form526['syncModern0781Flow'],
```

Comment:
Check for understanding: `form526['syncModern0781Flow']` will always be `true` if the above condition is met, correct?

Reply:
I think I asked you this same question... 
Yes I think in all cases the flipper is on, this key will be here and be `true`.  I think this was put here like this incase my understanding of that was not correct or just incase that ever changes.

Reply:
Understood. Thanks for confirming.